packages/moonstone/styles/variables.less

```diff
@@ -470,7 +470,7 @@
 @moon-notification-padding-left-right: 39px;
 @moon-notification-padding-top-bottom: 24px;
 @moon-notification-min-width: 300px;
-@moon-notification-max-width: 978px;
+@moon-notification-max-width: ((@moon-notification-border-width * 2) + (@moon-notification-padding-left-right * 2) + (2 * ((@moon-button-small-h-margin * 2) + @moon-button-max-width)));  // Attempt to calculate the width of all of the buttons and the pieces of Notification that influence the width to get the maximum size
```

Comment:
The math checks out but I'd suggest we use either the "small" variables or the "large" variables for button. My inclination would be the "large" values to ensure that two large buttons would fit (even though small buttons are the default).

Reply:
Does this account for large text mode, too?

Reply:
IIRC, large text doesn't affect the max width of buttons so we don't need to account for that here.